awx/ui_next/package-lock.json

```diff
@@ -220,7 +220,7 @@
         },
```

Comment:
We probably want to revert these changes so that we are using the secure url instead of the insecure 1.

Reply:
didn't realize I commited package-lock.json at all!  That was unintentional, I'll remove that.